packages/frontend/src/components/EditorLayout/EditorSnackbar.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Flex, Text } from '@chakra-ui/react'
+import { Button } from '@opengovsg/design-system-react'
+
+interface EditorSnackbarProps {
+  isOpen: boolean
+  handleUnpublish: () => void
+}
+
+export default function EditorSnackbar(props: EditorSnackbarProps) {
+  const { isOpen, handleUnpublish } = props
+  const snackbarText = 'To edit this pipe, you need to unpublish it first'
+  return (
+    <Box
+      position="fixed"
+      right="50%"
+      transform="translateX(50%)" // Center horizontally
```

Comment:
nit: not sure if this is the best way to centre an element horizontally. i think this is typically achieved with a combination of `display: flex` and `justify-content: center`

Reply:
i think i did it this way because the position is fixed and so the i couldn't center it with flex